modules/ROOT/pages/getting-started.adoc

```diff
@@ -11,7 +11,7 @@ include::_partials/attributes.adoc[]
 {org-name} {product-name} provides solutions to common development challenges faced by mobile developers.
 With OpenShift's Container technology providing a secure, scalable backend platform, mobile developers can use the {org-name} SDK which supports a range of native (iOS & Android) and hybrid (Xamarin & Cordova) platforms to create secure, high quality apps and makes it easy to add features such as Push Notifications or Authentication.
 
-IMPORTANT: {org-name} {product-name} is currently in a development phase. This makes it unsuitable for production deployment. Use this community release as an evaluation environment for developing and deploying mobile applications on Openshift.
```

Comment:
I believe that we're still documenting the community release here @laurafitzgerald 
which reminds me, how are we going to version this upcoming 'milestone' release?

Reply:
@finp resolving this now as we are close to final decision on versioning